### PR TITLE
Add config parsing tests for namers.

### PR DESCRIPTION
**Pull request by adleong**

### CODE REVIEW

**Comment by siggy** [comment]
lgtm :star: :+1:

File: linkerd/namer/fs/src/test/scala/io/l5d/FsTest.scala [hunk 1]
```diff
@@ -17,4 +20,20 @@ class FsTest extends FunSuite {
...
+      val yaml = s"""
+        |kind: io.l5d.fs
+        |rootDir: ${dir.getAbsolutePath}
+      """.stripMargin
+
+      val mapper = Parser.objectMapper(yaml)
+      FsInitializer.registerSubtypes(mapper)
+      val fs = mapper.readValue[NamerConfig](yaml).asInstanceOf[fs]
+      assert(fs.rootDir.path == dir.toPath)
+    } finally (Seq("echo", "rm", "-rf", dir.getPath).!)
```
Comment (klingerf) [inline linkerd/namer/fs/src/test/scala/io/l5d/FsTest.scala]: No "echo" here? Pretty sure this doesn't cleanup the test file as written.

**Comment by klingerf** [comment]
:star: :tada:

**Comment by olix0r** [comment]
:star: nice relaxing vacation work!

**Comment by klingerf** [comment]
:star: Thanks for updating.

**Pull request merged**